PlayerTest.py

```diff
@@ -0,0 +1,54 @@
+import player
+from tables import INFINITY
+import unittest
+
+
+class PlayerTest(unittest.TestCase):
+    def test_add_marbles(self):
+        player1 = player.Player('Glenn')
```

Comment:
Worthwhile to use a setup method (https://docs.python.org/2/library/unittest.html) that can define player1, player2 once for all players.